.github/workflows/score-checks.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           poetry run python3 data_pipeline/application.py score-full-run
       - name: Generate Score Post
         run: |
-          poetry run python3 data_pipeline/application.py generate-score-post -s local
+          poetry run python3 data_pipeline/application.py generate-score-post -s aws
```

Comment:
So this will automatically post a new download file and new tiles to the S3 bucket used by the site?

Reply:
No, this flag is to get the census data from aws. This doesn't upload anything to S3.